src/services/VirtualFileSystemService.spec.ts

```diff
@@ -1,532 +1,423 @@
-import { MetadataType } from './model';
+import path from 'path';
 import { VirtualFileSystemService } from './VirtualFileSystemService';
-import { VirtualDirectory } from './IVirtualFileSystemService';
+import { createTestContainer } from '../inversify.config';
 
-describe('FileSystemService', () => {
-  let service: VirtualFileSystemService;
+describe('VirtualFileSystemService', () => {
+  let virtualFileSystemService: VirtualFileSystemService;
```

Comment:
Wouldn't it be better to leave it as `service` in order not to introduce more differences between `FileSystemService.spec.ts` and `VirtualFileSystemService.spec.ts`?